### ISSUE TITLE
I'm able to pull/push to/from Trow but kubectl can't

### ISSUE THREAD

**Issue by tarikly** [ContainerSolutions/trow#283]
**Describe the bug**
I'm trying to get Trow working on my DigitalOcean k8s. I'am able to push/pull Trow from any client outside cluster and from nodes console but ``` kubectl trow-test trow.kube-public:3100/mynginx:test``` doesn't work.

**To Reproduce**

On DigitalOcean k8s I followed the Trow's manual install steps.

**Expected behavior**
I expect to have my kubectl trow-test up and running.

**Output/Logs**
I'am able to push/pull Trow from any client outside cluster and from k8s nodes console but ``` kubectl trow-test trow.kube-public:3100/mynginx:test``` gets ImagePullBackOff.

From my host I can run docker push/pull command without problems.

$ docker push trow.kube-public:31000/busybox:latest :
```
The push refers to repository [trow.kube-public:31000/busybox]
64cac9eaf0da: Layer already exists
latest: digest: sha256:50e44504ea4f19f141118a8a8868e6c5bb9856efa33f2183f5ccea7ac62aacc9 size: 527
```

$ docker pull trow.kube-public:31000/busybox:latest
```
latest: Pulling from busybox
3cb635b06aa2: Pull complete
Digest: sha256:50e44504ea4f19f141118a8a8868e6c5bb9856efa33f2183f5ccea7ac62aacc9
Status: Downloaded newer image for trow.kube-public:31000/busybox:latest
trow.kube-public:31000/busybox:latest
```
*Logs
$ kubectl -n kube-public logs --follow pod/trow-deploy-744888894b-8ck9b
```
2021-12-14T12:26:07 [rustls::session] ERROR TLS alert received: Message {
    typ: Alert,
    version: TLSv1_3,
    payload: Alert(
        AlertMessagePayload {
            level: Fatal,
            description: BadCertificate,
        },
    ),
}
```

$ kubectl -n kube-public describe pod trow-test

```
Name:         trow-test
Namespace:    default
Priority:     0
Node:         pool-dok8s-cluster-1-ug293/10.124.0.4
Start Time:   Tue, 14 Dec 2021 09:26:06 -0300
Labels:       run=trow-test
Annotations:  <none>
Status:       Pending
IP:           10.244.0.175
IPs:
  IP:  10.244.0.175
Containers:
  trow-test:
    Container ID:
    Image:          trow.kube-public:31000/busybox:latest
    Image ID:
    Port:           <none>
    Host Port:      <none>
    State:          Waiting
      Reason:       ErrImagePull
    Ready:          False
    Restart Count:  0
    Environment:    <none>
    Mounts:
      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-9vk5h (ro)
Conditions:
  Type              Status
  Initialized       True
  Ready             False
  ContainersReady   False
  PodScheduled      True
Volumes:
  kube-api-access-9vk5h:
    Type:                    Projected (a volume that contains injected data from multiple sources)
    TokenExpirationSeconds:  3607
    ConfigMapName:           kube-root-ca.crt
    ConfigMapOptional:       <nil>
    DownwardAPI:             true
QoS Class:                   BestEffort
Node-Selectors:              <none>
Tolerations:                 node.kubernetes.io/not-ready:NoExecute op=Exists for 300s
                             node.kubernetes.io/unreachable:NoExecute op=Exists for 300s
Events:
  Type     Reason     Age   From               Message
  ----     ------     ----  ----               -------
  Normal   Scheduled  15s   default-scheduler  Successfully assigned default/trow-test to pool-dok8s-cluster-1-ug293
  Normal   Pulling    14s   kubelet            Pulling image "trow.kube-public:31000/busybox:latest"
  Warning  Failed     14s   kubelet            Failed to pull image "trow.kube-public:31000/busybox:latest": rpc error: code = Unknown desc = failed to pull and unpack image "trow.kube-public:31000/busybox:latest": failed to resolve reference "trow.kube-public:31000/busybox:latest": failed to do request: Head "https://trow.kube-public:31000/v2/busybox/manifests/latest": x509: certificate signed by unknown authority
  Warning  Failed     14s   kubelet            Error: ErrImagePull
  Normal   BackOff    13s   kubelet            Back-off pulling image "trow.kube-public:31000/busybox:latest"
  Warning  Failed     13s   kubelet            Error: ImagePullBackOff
```

**Trow Info**
 - Install method: Manual install (https://github.com/ContainerSolutions/trow/blob/main/quick-install/MANUAL_INSTALL.md)
 - Version/Image Name: containersol/trow:default

**Kubernetes**
 - Kubernetes distro/host: DigitalOcean K8s
 - Kubernetes version: 1.21.5-do.0
 - Container Runtime: containerd://1.4.11

**Comment by ariyonaty** [comment]
Hey there, 

I recently stumbled upon this as well. 
The solution was to use a Kubernetes version prior to 1.20.X. 

Reasoning behind this, and as mentioned in the [DOKS changelog](https://docs.digitalocean.com/products/kubernetes/changelog/), is that _"Starting with DigitalOcean Kubernetes 1.20, containerd is used as the container runtime. Previous releases used Docker."_ 

Next, referencing this [issue](https://github.com/ContainerSolutions/trow/issues/78), it is mentioned that the quick-install you went through does not support the containerd runtime. (Though it does sound like containerd 1.4 solves the issue regarding copying certs..) 

Once reverting to v1.19.X, I was able to successfully deploy trow inside of the kubernetes cluster. 

Hope this clears things up.
